e2e/pluginFixtures.js

```diff
@@ -55,25 +53,22 @@ const path = require('path');
  * @param {ObjectCreateOptions} options
  * @returns {Promise<string>} uuid of the domain object
  */
-// async function getOrCreateDomainObject(page, options) {
-//     const { type, name } = options;
-//     const objectName = name ? `${type}:${name}` : type;
+async function getOrCreateDomainObject(page, options) {
```

Comment:
Are we going to be using this? Last comment I had on this line was to remove it.

Reply:
Let me remove it